app/repositories/cinema_hall_repository.rb

```diff
@@ -0,0 +1,17 @@
+module Repositories
+    class CinemaHallRepository
+        attr_reader :adapter
+
+        def initialize(adapter: Models::CinemaHall)
+            @adapter = adapter
+        end
+
+        def fetch_all_with_columns(columns:)
+            adapter.select(*columns)
+        end
+
+        # def find_all
+        #     adapter.all
+        end
```

Comment:
I am afraid it's a syntax error (incorrect code, I see `end` but I don't see a 'def` - it's commented.

Generally, it's really bad to keep commented code. It's not the purpose of comments.

https://kentcdodds.com/blog/please-dont-commit-commented-out-code